README.md

```diff
@@ -1,38 +1,50 @@
-
-Because Mason is developed by Mapbox the default S3 bucket and set of available packages are driven by Mapbox developers. For other developers: 1) fork Mason and ideally rename it to something like `mason-{yourorg}`, 2) configure it against your own s3 bucket, and 3) go ahead an publish your own packages to an S3 bucket of your creation.
+Mason is unlike:
 
-### Comparisons
+ * all of the above...
 
-Mason is like [npm](https://github.com/npm/npm) because it installs packages in the current working directory (`./mason_packages`) rather than globally.
+    Mason is a collection of bash scripts and does not depend on any specific runtime language, such as python, node.js, or ruby. It can build and publish a single set of binaries (>= OS X 10.8 and >= Ubuntu Precise), publish header-only files, and install packages. Mason has integrations with [Travis CI](https://travis-ci.org) and [Amazon S3](https://aws.amazon.com/s3) for automated build and deployment.
 
-Mason is like [homebrew](http://brew.sh/) because it requires no use of `sudo` to install packages.
+    Mason strongly prefers static libraries over shared libraries and has zero understanding of dependency trees: it leaves complete control to the developer for how packages relate.
 
-Mason is like linux package managers like `apt-get` or `yum` because it works on linux.
+Mason works on both **OS X** and **Linux**.
 
-Mason is unlike all of the above package managers because:
+[![Build Status](https://travis-ci.org/mapbox/mason.svg?branch=master)](https://travis-ci.org/mapbox/mason)
 
-  - Mason runs on both Linux and OS X and creates a single set of binaries that work on >= OS X 10.8 and >= Ubuntu Precise (rather than building binaries per version).
-  - Mason strongly prefers static libraries over shared libraries
-  - Mason has zero understanding of dependency trees: it leaves complete control to the developer for how packages relate.
-  - Mason does not depend on any specific runtime language (like python, node.js or ruby). It is a just a few bash scripts.
-  - Mason depends on [travis.ci](https://travis-ci.org) for creating and publishing binaries.
+# Table of Contents
+
+- [Installation](#installation)
+    + [Curl install](#curl-install)
+    + [Submodule](#submodule)
+    + [mason.cmake](#masoncmake)
+- [Configuration](#configuration)
+- [Usage](#usage)
+- [Creating a package](#creating-a-package)
+    * [Prerequisites](#prerequisites)
+    * [Getting started](#getting-started)
+    * [Script structure](#script-structure)
+    * [Mason variables](#mason-variables)
+    * [Mason functions](#mason-functions)
+    * [System packages](#system-packages)
+- [Troubleshooting](#troubleshooting)
+- [Conventions](#conventions)
 
 ## Installation
 
-There are three recommended ways to install mason: via curl, via a submodule, or via bundling `mason.cmake`
+There are three recommended ways to install mason, via:
+
+    * curl
```

Comment:
Looks like adding tabs automatically converts to a code block format
```
like this
```

Not sure if that was intended. Bullet points in markdown automatically indent.

Reply:
p.s. Love how this intros and sets the stage for the three ways to install

Reply:
Not intended. Fixed!